extensions/wikia/UserLogin/EmailConfirmationHooks.class.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+class EmailConfirmationHooks {
+
+	/**
+	 * Add JS messages to the output
+	 * @param \OutputPage $out An output object passed from a hook
+	 * @return bool
+	 */
+	public static function onBeforePageDisplay( \OutputPage $out ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$out". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)